js/interface.js

```diff
@@ -555,6 +571,217 @@ function uploadFiles(files) {
   });
 }
 
+// FUNCTIONS FOR MOVING FILES/FOLDERS IN MODAL
+
+// Check if folder that has selected items to move has a subfolder and filter it
+function excludeFolders(items) {
+  var parentItemIdOfSelectedItems = navStack[navStack.length-1].id;
+  var selectedIdPlaceInModal = foldersForMoving.navStack[foldersForMoving.navStack.length-1].id;
+  var searchResult = items;
```

Comment:
Result is singular, but you're assigning a variable called items (plural). Which is correct?

Reply:
fixed to plural name